Menu/Menu.js

```diff
@@ -1,10 +1,18 @@
 
 const toggleMenu = () => {
-  // Toggle the "menu--open" class on your menu refence. 
+  var element = document.querySelector(".menu");
+
+  element.classList.toggle('menu');
```

Comment:
You want to toggle with another class name, not the same one you selected the element with.